awx/ui/client/src/instance-groups/add-edit/instance-list-policy.controller.js

```diff
@@ -0,0 +1,56 @@
+function InstanceModalController ($scope, $state, $http, $q, models, strings) {
```

Comment:
I think you can probably remove the `$http` and `$q` dependencies - they don't seem to be used anywhere?